bin/marathon-framework

```diff
@@ -72,7 +72,13 @@ function load_options_and_log {
     cmd+=( "$env_op" )
   done
   echo "${cmd[@]}"
-  logged marathon "${cmd[@]}" "$@"
+
+  if [[ "${cmd[@]} $@" == *'--no-logger'* ]]
+  then
+    marathon "${cmd[@]}" "$@"
```

Comment:
@sielaq thanks for the PR. Can you please describe this change? I haven't run it but at first glance I don't think it's working the way it's supposed to. If it runs anything then probably because an executable called "marathon" is somewhere in your $PATH. I think the string "marathon" is too much here as it's not a function and in the context of "logged marathon" the string is only used as the logger name.